kernel/make_initrd.sh

```diff
@@ -23,7 +23,7 @@ cp -va $BUSYBOX/busybox $TEMP/bin
 cd $TEMP
 mkdir dev proc sys tmp sbin
 mknod dev/console c 5 1
-cat > $TEMP/init <<EOF
```

Comment:
Without quotes interpreter tires to evaluate all $(...) inside <<EOF ... EOF. 
Example:

```
cat > some.txt <<EOF
echo $(uname)
EOF

File content:
Linux
```

```
cat > some.txt <<'EOF'
echo $(uname)
EOF

File content:
echo $(uname)
```